### PR TITLE
🌐: Add Russian Translations

### PR DESCRIPTION
**Pull request by apxgpt**
These are not significant changes; I added some translation for the Russian language and included translations for the buttons:
Continue
Regenerate
Stop

### CODE REVIEW

**Review by berry-13 — APPROVED** [review]
I hadn't noticed them, thank you!

File: client/src/localization/languages/Eng.tsx [hunk 1]
```diff
@@ -79,6 +80,7 @@ export default {
   com_auth_password_not_match: 'Passwords do not match',
   com_auth_continue: 'Continue',
   com_auth_create_account: 'Create your account',
+  com_auth_description_create_account: 'Note that phone verification may be required for signup. Your number will only be used to verify your identity for security purposes.',
```
Comment (berry-13) [inline client/src/localization/languages/Eng.tsx]: Why did you add this translation that is not used anywhere?

**Review by berry-13 — CHANGES_REQUESTED** [review]
I apologize, I was on mobile and couldn't see the changes properly...

1. Add the English translation for `com_ui_continue` and `com_ui_regenerate`.
2. To resolve the problem with the tests, replace in `Continue.spec`, `Regenerate.spec`, and `Stop.spec` the line `import { render, fireEvent } from @testing-library/react` with `import { render, fireEvent } from 'test/layout-test-utils';`

File: client/src/localization/languages/Eng.tsx (hunk 1 above)
Comment (apxgpt) [reply client/src/localization/languages/Eng.tsx]: I am using com_auth_description_create_account in my code, but I haven't finished it yet. I will remove it for you.

Comment (berry-13) [reply client/src/localization/languages/Eng.tsx]: Have you by any chance added the ability to use SMS for verification?

Comment (apxgpt) [reply client/src/localization/languages/Eng.tsx]: I accidentally added additional description, but you don't have it, so it's unnecessary for your repository.

**Review by berry-13 — APPROVED** [review]
Okay, sorry for the mess I made with the reviews. Thanks again!
